CI Broken: spawn_robot (multirobot test)

All CI is failing on a change to the test bringups getting XML path & not being able to find a spawn_robot package:


```
15: [spawn_robot-2] Traceback (most recent call last):
15: [spawn_robot-2]   File "/opt/overlay_ws/install/spawn_robot/lib/spawn_robot/spawn_robot", line 11, in <module>
15: [spawn_robot-2]     load_entry_point('spawn-robot', 'console_scripts', 'spawn_robot')()
15: [spawn_robot-2]   File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 489, in load_entry_point
15: [spawn_robot-2]     return get_distribution(dist).load_entry_point(group, name)
15: [spawn_robot-2]   File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 2852, in load_entry_point
15: [spawn_robot-2]     return ep.load()
15: [spawn_robot-2]   File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 2443, in load
15: [spawn_robot-2]     return self.resolve()
15: [spawn_robot-2]   File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 2449, in resolve
15: [spawn_robot-2]     module = __import__(self.module_name, fromlist=['__name__'], level=0)
15: [spawn_robot-2] ModuleNotFoundError: No module named 'spawn_robot'
15: [ERROR] [spawn_robot-2]: process has died [pid 5207, exit code 1, cmd '/opt/overlay_ws/install/spawn_robot/lib/spawn_robot/spawn_robot --robot_name robot1 --robot_namespace robot1 --sdf /opt/overlay_ws/src/navigation2/nav2_system_tests/models/turtlebot3_waffle/model.sdf -x 0.0 -y 0.5 -z 0.01 --ros-args'].
15: [controller_server-9] [INFO] [robot1.controller_server]: Created local controller : dwb_core::DWBLocalPlanner
15: [spawn_robot-3] Traceback (most recent call last):
15: [spawn_robot-3]   File "/opt/overlay_ws/install/spawn_robot/lib/spawn_robot/spawn_robot", line 11, in <module>
15: [spawn_robot-3]     load_entry_point('spawn-robot', 'console_scripts', 'spawn_robot')()
15: [spawn_robot-3]   File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 489, in load_entry_point
15: [spawn_robot-3]     return get_distribution(dist).load_entry_point(group, name)
15: [spawn_robot-3]   File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 2852, in load_entry_point
15: [spawn_robot-3]     return ep.load()
15: [spawn_robot-3]   File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 2443, in load
15: [spawn_robot-3]     return self.resolve()
15: [spawn_robot-3]   File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 2449, in resolve
15: [spawn_robot-3]     module = __import__(self.module_name, fromlist=['__name__'], level=0)
15: [spawn_robot-3] ModuleNotFoundError: No module named 'spawn_robot'
```

This commit https://github.com/ros-planning/navigation2/pull/1269/commits/a8fece364864c3a32758d32fba758eabfe37bcf4 fixes the XML BT paths, but the spawn_robot error is still around

--- a/nav2_system_tests/src/system/test_multi_robot_launch.py
+++ b/nav2_system_tests/src/system/test_multi_robot_launch.py
@@ -17,7 +17,7 @@
 import os
 import sys
 
-from ament_index_python.packages import get_package_prefix, get_package_share_directory
+from ament_index_python.packages import get_package_share_directory
 
 from launch import LaunchDescription, LaunchService
 from launch.actions import (ExecuteProcess, GroupAction,
@@ -35,7 +35,7 @@ def generate_launch_description():
     urdf = os.getenv('TEST_URDF')
     sdf = os.getenv('TEST_SDF')
 
-    bt_xml_file = os.path.join(get_package_prefix('nav2_bt_navigator'),
+    bt_xml_file = os.path.join(get_package_share_directory('nav2_bt_navigator'),
                                'behavior_trees',
                                os.getenv('BT_NAVIGATOR_XML'))
 

--- a/nav2_system_tests/src/system/test_system_launch.py
+++ b/nav2_system_tests/src/system/test_system_launch.py
@@ -17,7 +17,7 @@
 import os
 import sys
 
-from ament_index_python.packages import get_package_prefix, get_package_share_directory
+from ament_index_python.packages import get_package_share_directory
 
 from launch import LaunchDescription
 from launch import LaunchService
@@ -33,7 +33,7 @@ def generate_launch_description():
     map_yaml_file = os.getenv('TEST_MAP')
     world = os.getenv('TEST_WORLD')
 
-    bt_navigator_xml = os.path.join(get_package_prefix('nav2_bt_navigator'),
+    bt_navigator_xml = os.path.join(get_package_share_directory('nav2_bt_navigator'),
                                     'behavior_trees',
                                     os.getenv('BT_NAVIGATOR_XML'))
 

